File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/MedicalInformation.cs
using System;
using System.Collections.Generic;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    public class MedicalInformation
    {
        private String _medicalClass;
        private String _medicalClassName;
        private String _medicalClassNumber;
        private List<MedicationInfo> _medicationInfo = new List<MedicationInfo>();

        public String MedicalClass
        {
            get { return this._medicalClass; }
            set { this._medicalClass = value; }
        }

        public String MedicalClassName
        {
            get { return this._medicalClassName; }
            set { this._medicalClassName = value; }
        }

        public String MedicalClassNumber
        {
            get { return this._medicalClassNumber; }
            set { this._medicalClassNumber = value; }
        }

        public List<MedicationInfo> MedicationInfo
        {
            get { return this._medicationInfo; }
            set { this._medicationInfo = value; }
        }

        public MedicalInformation() { }

        public MedicalInformation(String medicalClass, String medicalClassName, String medicalClassNumber, List<MedicationInfo> medicationInfo)
        {
            this._medicalClass = medicalClass;
            this._medicalClassName = medicalClassName;
            this._medicalClassNumber = medicalClassNumber;
            this._medicationInfo = medicationInfo;
        }

    }
}

File: MED-CLOUD/med-ext/med-ext-orca/src/main/java/nta/med/orca/gw/api/contracts/message/MedicalInformation.java
package nta.med.orca.gw.api.contracts.message;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;

import java.util.List;

/**
 * @author dainguyen.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class MedicalInformation {

    private String medicalClass;
    private String medicalClassName;
    private String medicalClassNumber;
    private List<MedicationInfo> medicationInfos;

    @JacksonXmlProperty(localName = "Medical_Class")
    public String getMedicalClass() {
        return medicalClass;
    }

    public void setMedicalClass(String medicalClass) {
        this.medicalClass = medicalClass;
    }

    @JacksonXmlProperty(localName = "Medical_Class_Name")
    public String getMedicalClassName() {
        return medicalClassName;
    }

    public void setMedicalClassName(String medicalClassName) {
        this.medicalClassName = medicalClassName;
    }

    @JacksonXmlProperty(localName = "Medical_Class_Number")
    public String getMedicalClassNumber() {
        return medicalClassNumber;
    }

    public void setMedicalClassNumber(String medicalClassNumber) {
        this.medicalClassNumber = medicalClassNumber;
    }

    @JacksonXmlProperty(localName = "Medication_info")
    public List<MedicationInfo> getMedicationInfos() {
        return medicationInfos;
    }

    public void setMedicationInfos(List<MedicationInfo> medicationInfos) {
        this.medicationInfos = medicationInfos;
    }
}
